test/connectedGraph_test.js

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai'
+import connectedGraph from '../src/connectedGraph'
+
+describe('connectedGraph()', function(){
+
+  it('should be a function', function(){
+    expect(connectedGraph).to.be.a('function')
+  })
+
+  it('returns weather or not all points are connected', function(){
+    const graphA = {
+      'a': ['b', 'c'],
+      'b': ['a', 'd'],
+      'c': ['a', 'd'],
+      'd': ['b', 'c']
+    }
+    const graphB = {
+      'a': ['b'],
+      'b': ['a'],
+      'c': ['d'],
+      'd': ['c']
+    }
+
+    let a = connectedGraph(graphA)
+    let b = connectedGraph(graphB)
+    expect(a).to.equal(true)
+    expect(b).to.equal(false)
+  })
+})
```

Comment:
Maybe a couple more test case's would help someone reading this test fully understand what this function does.